### PR TITLE
Renamed convex hull section to just geometry

### PR DESCRIPTION
**Pull request by dolphingarlic**

### CODE REVIEW

**Review by bqi343 — CHANGES_REQUESTED** [review]
description should be changed (previous description referred to nearly all platinum geometry problems being convex hull)

File: content/ordering.ts [hunk 1]
```diff
@@ -176,8 +176,8 @@ const MODULE_ORDERING: { [key in SectionID]: Chapter[] } = {
       items: ['bin-jump', 'merging', 'hld', 'centroid'],
     },
     {
-      name: 'Convex Hull',
-      description: 'Most Platinum geometry problems.',
+      name: 'Geometry',
+      description: 'More advanced concepts in computational geometry.',
```
Comment (bqi343) [inline content/ordering.ts:180]: > More advanced

more advanced than what? Geometry hasn't appeared earlier in the guide :)

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/ordering.ts (hunk 1 above)
Comment (dolphingarlic) [reply content/ordering.ts:180]: There's rectangle geometry in bronze

Comment (bqi343) [reply content/ordering.ts:180]: fair enough

**Review by bqi343 — APPROVED** [review]

**Pull request merged**